rest/src/main/scala/io/udash/rest/RestDataCompanion.scala

```diff
@@ -109,3 +110,38 @@ abstract class RestDataWrapperCompanion[Wrapped, T](implicit
     (resolver: SchemaResolver, schemaTransform: RestSchema[_] => RestSchema[_]) =>
       wrappedResponses.responses(resolver, ws => schemaTransform(nameAndAdjusters.restSchema(ws)))
 }
+
+/**
+  * Base class for companion objects of enum types [[ValueEnum]] which are used as
+  * parameter or result types in REST API traits. Automatically provides instance of
+  * [[io.udash.rest.openapi.RestSchema RestSchema]].
+  */
+abstract class RestValueEnumCompanion[E <: ValueEnum](implicit
+  instances: MacroInstances[DefaultRestImplicits, () => NameAndAdjusters[E]]
+) extends AbstractValueEnumCompanion[E] {
+  implicit lazy val restSchema: RestSchema[E] =
+    RestValueEnumCompanion.standaloneNamedEnumSchema[E](this, instances(DefaultRestImplicits, this).apply())
+}
+object RestValueEnumCompanion {
```

Comment:
Do we need this separate object? Does not seem consistent with the rest of the API / companions. Should we move this method to `io.udash.rest.openapi.RestSchema` instead or just inline?

Reply:
It can be inlined, done